packages/veritone-widgets/src/redux/modules/mediaDetails/faceEngineOutput/index.js

```diff
@@ -211,19 +224,60 @@ const reducer = createReducer(defaultState, {
       facesRemovedByUser: {}
     };
   },
-  [OPEN_CONFIRMATION_DIALOG](state, action) {
-    const { confirmationAction } = action.payload;
+  [OPEN_CONFIRMATION_DIALOG](
+    state,
+    {
+      payload: { confirmationType }
+    }
+  ) {
     return {
       ...state,
       showConfirmationDialog: true,
-      confirmationAction: confirmationAction || noop
+      confirmationType: confirmationType || 'cancelEdits'
     };
   },
-  [CLOSE_CONFIRMATION_DIALOG](state, action) {
+  [CLOSE_CONFIRMATION_DIALOG](state) {
     return {
       ...state,
       showConfirmationDialog: false,
-      confirmationAction: noop
+      confirmationType: 'cancelEdits'
+    };
+  },
+  [SAVE_FACE_EDITS](state) {
+    return {
+      ...state,
+      savingFaceEdits: true
+    };
+  },
+  [SAVE_FACE_EDITS_SUCCESS](state) {
+    return {
+      ...state,
+      error: null,
```

Comment:
You should declare 'error: null' in the default state.

Reply:
done